README_RU.md

```diff
@@ -0,0 +1,56 @@
+[English ](README.md) | [Русский ](README_RU.md)
```

Comment:
I want to keep README to be English only because it would be hard to update all languages at once whenever I want to update README.